src/ceph-volume/ceph_volume/tests/test_inventory.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+from ceph_volume.util.device import Devices
+from ceph_volume import sys_info
+
+class TestInventory(object):
+
+    def test_json_inventory(self):
```

Comment:
I restructured the test a bit. Since fixtures and the parametrize decorator do not yet play well together I kept the looped assert. I don't think in this case it's crucial to catch all unexpected or missing keys in one go.